### DISCUSSION TITLE
Logs file

### DISCUSSION THREAD

**Discussion by colladoscl** [Seji64/LAPS-WebUI#21] [General]
First, thanks for the project, it works perfectly.

I would like to have logs of who uses de app, just to know who consults which computer. 

I don`t know if this can be done using some config params like the written below in appsetting.json, I´ve tried but it doesn´t work.

  "Logging": {
    "LogLevel": {
      "Default": "Debug",
      "Microsoft": "Warning",
      "Microsoft.Hosting.Lifetime": "Information"
    },
  "Path": { 
	"LogFilePath": "D:\\LAPS-WebUI-v1.5.6-win-x64\\logs\\"
	}
  },

Is there anyway to get it done changing some configs?

Thanks

**Comment by Seji64** [comment]
Hi,

Sorry no, there are currently no access Logs. So there is no way to enable them via configuration.

**Comment by Seji64** [comment]
Hi,
please try the new version and see https://github.com/Seji64/LAPS-WebUI#-logging-to-file-serilog
